docs/installation.rst

```diff
@@ -0,0 +1,118 @@
+.. _installing_ansible_navigator:
+
+Installing `ansible-navigator` with execution environment support
+#################################################################
+
+.. contents::
+   :local:
+
+Linux
+*****
+
+Requirements
+============
+
+* Either `podman` or `Docker for Linux`
+* Internet access (during initial installation)
+
+Installing on Linux with `podman`
+=================================
+
+* Follow the `podman` installation instructions for the appropriate `Linux distribution <https://podman.io/getting-started/installation>`
+
+Install on Linux with `Docker for Linux`
+========================================
+
+* Follow the `Docker for Linux` installation instructions for the appropriate `Linux distribution <https://docs.docker.com/engine/install/>`
+
+Install `ansible-navigator`
```

Comment:
In RST, single backticks have different semantics than in MD. You'd have to change every place with double backticks with this transition.

Reply:
Now you need to fix all those 67 warnings to make the docs build to succeed: https://readthedocs.org/projects/ansible-navigator/builds/15280151/.